lib/pipeline/statelessPipelineStack.ts

```diff
@@ -26,10 +28,26 @@ export class StatelessPipelineStack extends cdk.Stack {
     const ghRunnerRole = new iam.Role(this, 'GHRunnerRole', {
       assumedBy: new iam.ServicePrincipal('codebuild.amazonaws.com'),
     });
+    const name = 'orcabus-codebuild-gh-runner';
+
+    // Based on https://github.com/aws/aws-cdk/blob/2b2443de2f566f1595657f94195d8b61243fb800/packages/aws-cdk-lib/aws-codebuild/lib/project.ts#L1328-L1342
+    const logGroupArn = Stack.of(this).formatArn({
+      service: 'logs',
+      resource: 'log-group',
+      arnFormat: ArnFormat.COLON_RESOURCE_NAME,
+      resourceName: `/aws/codebuild/${name}`,
+    });
+    ghRunnerRole.addToPolicy(
+      new PolicyStatement({
+        resources: [logGroupArn, `${logGroupArn}:*`],
+        actions: ['logs:CreateLogGroup', 'logs:CreateLogStream', 'logs:PutLogEvents'],
+      })
+    );
```

Comment:
I'm not certain that this is correct, is there a way I can test it?

Otherwise, I could also just allow on all resources using `*`.

Reply:
I think It's about right.

Reply:
I suppose the `effect` here is default to allow?

Reply:
Yeah, should be defaulting to allow, although it's undocumented: https://github.com/aws/aws-cdk/blob/53dc0d81e004ad1f8513af534b6d43f4315c24a3/packages/aws-cdk-lib/aws-iam/lib/policy-statement.ts#L90C19-L90C22